### PR TITLE
TAOS-CI: Updated the configuraiton files

### PR DESCRIPTION
**Pull request by leemgs**
New version of TAOS-CI is released on Mar-24-2020.
* Release note - https://github.com/nnsuite/TAOS-CI/releases/tag/v1.5.20200324

This commit is to update the existing configuration files of the nnstreamer repository.

**Changelog**
  * Updated configuration files
  * Updated README.md file
  * Removed unnecessary files (Deprecated)

Signed-off-by: Geunsik Lim <geunsik.lim@samsung.com>

### CODE REVIEW

File: .TAOS-CI/README.md [hunk 1]
```diff
@@ -1,13 +1,51 @@
...
 - http://github.com/nnsuite/TAOS-CI
 
-## How to use
+## How to use the configruation files
+The section describes two steps to use the configuration files of the nnstreamer repository.
 
-After adding TAOS-CI submodule in your project, copy and overwrite them.
+#### Step 1
+First of all, you must write for passwords carefully as follows.
+!!!SECURITY!!! NEVER REVEAL THE BELOW PASSWORDS BELOW.
```
Comment (wooksong) [inline .TAOS-CI/README.md]: There is a duplicate word, 'below'.

```suggestion
!!!SECURITY!!! NEVER REVEAL THE BELOW PASSWORDS.
```

**Comment by myungjoo** [comment]
If we apply this, could TAOS-CI will load these files?

(Will TAOS_CI change behavior when we update & merge these files? I want such behaviors; I want to update TAOS-CI behaviors by updating files in nnstreamer.git)

**Comment by leemgs** [comment]
> If we apply this, could TAOS-CI will load these files?

Absolutely. 
* NOTE: Instead, never update & merge the passwords of the configuration files  with git for security.

File: .TAOS-CI/README.md (hunk 1 above)
Comment (leemgs) [reply .TAOS-CI/README.md]: Nice catch. Fixed.

**Comment by leemgs** [comment]
To All, 

This PR will be re-submmited newly after closing due to the incorrect branch structure.

**Pull request closed without merging**